CIPs/CIP-59/CIP-59.md

```diff
@@ -48,15 +47,12 @@ The multicodec for DocID is `0xce`
 For compatibility with browser urls it's recommended to encode the DocID using [`base36`](https://github.com/multiformats/multibase).
 
 ### Registered values
-Using the tables linked below doctypes and subnets can be be publicly registered by submitting a new CIP that adds it to one of the tables.
+Using the table linked below doctypes can be be publicly registered by submitting a new CIP that adds it to the table.
 
 * [Doctypes table](./tables/doctypes.csv)
-* [Subnets table](./tables/subnets.csv)
 
 ## Rationale
-A Ceramic document can be identified using the CID of the *genesis record* of the document log, as well as the subnet and doctype of the document. Using the DocID a Ceramic node can query the network for the latest *tip* of the document. 
-
-The subnet is used to uniquely identify a subnet on the Ceramic network. Initially this is used to determine if a document has been created on the testnet (`0`) or the livenet (`1`). In the future more subnets may be defined.
+A Ceramic document can be identified using the CID of the *genesis record* of the document log, as well as the doctype of the document. Using the DocID a Ceramic node can query the network for the latest *tip* of the document. 
```

Comment:
Interesting.  This means that two documents could theoretically share the same genesis record if they have different doctypes.  I guess that's probably fine?

Reply:
Yeah, I don't see any issue with that.